plugin/src/software/aws/toolkits/eclipse/amazonq/lsp/manager/DefaultLspManager.java

```diff
@@ -238,6 +247,15 @@ private static void makeExecutable(final Path filePath) throws IOException {
                 PosixFilePermission.OTHERS_READ, PosixFilePermission.OTHERS_EXECUTE));
         Files.setPosixFilePermissions(filePath, permissions);
     }
+    private static void showDeprecatedManifestNotification() {
```

Comment:
How do we handle the case when a user closes the notification? Will it continue popping up every time it launches or what is the condition to show this?
This should not be a modal prompt and interrupt the user's flow

Reply:
The notification appears as a small pop up over the legal text in the Q window.  Doesn't pull the users cursor and disappears on its own after a handful of seconds. 
In terms of the condition, currently it will show if the users loaded manifest is marked as deprecated; it would show on each launch. If preference is to show only once I'll add flag to the plugin store.

Reply:
I was asking because some users might intentionally choose to keep this version installed. Notification every time they launch might be too aggressive. In situations like this we usually have a Dont show again button that users can click and then on subsequent launches we honor that and dont show the notification again